pkg/build/gobuild.go

```diff
@@ -544,6 +548,13 @@ func tarBinary(name, binary string, platform *v1.Platform) (*bytes.Buffer, error
 		header.PAXRecords = map[string]string{
 			"MSWINDOWS.rawsd": userOwnerAndGroupSID,
 		}
+		header.Format = tar.FormatPAX
```

Comment:
Not sure if this format is necessary for Windows, but it seemed like I had to overwrite it on linux.